x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -16384,7 +16384,7 @@
     "xpack.securitySolution.endpoint.resolver.panel.processEventListByType.eventDescriptiveName": "{descriptor} {subject}",
     "xpack.securitySolution.endpoint.resolver.panel.processEventListByType.events": "イベント",
     "xpack.securitySolution.endpoint.resolver.panel.processEventListByType.wait": "イベントを待機しています...",
-    "xpack.securitySolution.endpoint.resolver.panel.processListWithCounts.events": "すべてのプロセスイベント",
```

Comment:
:information_source: Was this automated or did you do this?

Reply:
i did it. turns out changing those things is a pain now.

Reply:
are they going to need to be manually changed from now on then?

Reply:
Yeah, this seems like a thing we should cover as a team. 1) What do you do when you change the actual semantics of a translation? Like changing "Terminated Process" to "Terminated Trigger" or something? Do you delete the label and make a new one? 2) Is it safe to reuse these if the semantics are the same?

Reply:
good point. not sure. 
Related, we could talk as a team about how to pick IDs.